storage/google/cloud/storage/batch.py

```diff
@@ -200,9 +206,11 @@ def _prepare_batch_request(self):
 
         multi = MIMEMultipart()
 
-        for method, uri, headers, body in self._requests:
+        timeout = None
+        for method, uri, headers, body, timeout in self._requests:
             subrequest = MIMEApplicationHTTP(method, uri, headers, body)
             multi.attach(subrequest)
+            timeout = timeout
```

Comment:
Nit: Could one of the `timeout`s be renamed? I had a little trouble figuring out what was happening here.

Reply:
oh, lord. My bad. Let me fix that. And my bad also, didn't run coverage locally.

Reply:
Related: I am defaulting to the timeout of the last request added to the batch. I could just as well see adding the timeouts up for each request in batch. Either seems like a fair enough approach.